Source/WebCore/crypto/cocoa/CryptoAlgorithmHMACMac.cpp

```diff
@@ -71,4 +89,25 @@ ExceptionOr<bool> CryptoAlgorithmHMAC::platformVerify(const CryptoKeyHMAC& key,
     return signature.size() == expectedSignature.size() && !constantTimeMemcmp(expectedSignature.data(), signature.data(), expectedSignature.size());
 }
 
+ExceptionOr<Vector<uint8_t>> CryptoAlgorithmHMAC::platformSign(const CryptoKeyHMAC& key, const Vector<uint8_t>& data, UseCryptoKit useCryptoKit)
+{
+#if HAVE(SWIFT_CPP_INTEROP)
+    if (useCryptoKit == UseCryptoKit::Yes)
+        return platformSignCryptoKit(key, data);
+#else
+    UNUSED_PARAM(useCryptoKit);
```

Comment:
This looks like an opportunity to log. Or are there valid cases where we'd get `UseCryptoKit::Yes` but `HAVE(SWIFT_CPP_INTEROP)` is false?

Reply:
Could even assert if it's really not supported.